CWE/events/italy.txt

```diff
@@ -1899,6 +1903,237 @@ country_event = {
 
 }
 
+#Death of Enrico Mattei
+country_event = {
+  id = 8020329
+  title = "The Death of Enrico Mattei"
+  desc = EVTDESC8020329
+  picture = "enrico_mattei"
+  
+  fire_only_once = yes
+
+	trigger = {
+		tag = ITA
+		year = 1962
+		NOT = { year = 1990 }
+		
+		has_country_flag = ITA_Enrico_Mattei
+	}
+
+	mean_time_to_happen = { months = 10 }
+
+	option = {
+		name = "Suspicious..."
+		prestige = -5
+		
+		any_pop = { 
+			limit = { strata = rich }
+			consciousness = -4
+		}
+	}
+
+}
+
+#1963 elections Historical
+country_event = {
+	id = 8020330
+	title = "$YEAR$ Italian General Election"
```

Comment:
Please insert check to make sure Italy is a parliamentary republic for all historical election events